frontend/packages/operator-lifecycle-manager/integration-tests-cypress/tests/operator-install-single-namespace.spec.ts

```diff
@@ -58,42 +58,31 @@ describe(`Interacting with a single namespace install mode Operator (${operatorN
     cy.byTestID('view-installed-operators-btn').click();
     cy.log(`verify the ClusterServiceVersion row for ${operatorRow} exists`);
     cy.byTestOperatorRow(operatorRow, { timeout: 60000 }).should('exist');
-  });
-
-  it(`displays details about ${operatorName} ClusterServiceVersion on the "Details" tab`, () => {
```

Comment:
In the global test, you migrated these to a `cy.log`, which I like.  Do the same here?

Reply:
Yes, but in most cases the `it` was followed immediately by a `cy.log`, ie:
```
it(`displays empty message on the ${operatorName} ClusterServiceVersion "All Instances" tab`, () => {
    cy.log('navigate to the "All Instances" tab');
```
In these cases, I tended to use the `cy.log`, because it was concise, whereas the `it` I feel is verbose, and a new overall `it` had been created.

Reply:
If that's the case, you don't need `operatorInstance` as a `const` since it is only used once.

Reply:
fixed, ty